hooks/src/test/useCardCVC.test.ts

```diff
@@ -0,0 +1,71 @@
+import { ChangeEvent } from "react";
+import { renderHook, act } from "@testing-library/react";
+import { COMMON_ERROR, CVC_NUMBER } from "../lib/constants/validation";
+import useCardCVC from "../lib/useCardCVC";
+
+describe("useCardCVC 커스텀 훅 테스트", () => {
+  const cardCVCLength = 3;
+
+  it("입력된 CVC가 전부 숫자가 아닌 경우 isValid는 false이고, 숫자를 입력해달라는 에러 메세지를 출력한다.", () => {
+    const CVCNotOnlyNumber = "12A";
+    const { result } = renderHook(() => useCardCVC(cardCVCLength));
+
+    act(() => {
+      result.current.handleCardCVC({
+        target: { value: CVCNotOnlyNumber },
+      } as ChangeEvent<HTMLInputElement>);
+    });
```

Comment:
중요한 부분은 아니지만 이렇게 정리해도 더 가독성이 좋지 않을까 하는 생각이 들었습니다!
```suggestion

    const invalidEvent = { target: { value: CVCNotOnlyNumber } } as ChangeEvent<HTMLInputElement>);
    act(() => {
      result.current.handleCardCVC(invalidEvent);
```

Reply:
이렇게 과정을 한 번 쪼개는 게 테스트에서 어떤 작업을 하고 있는지 더 잘 보이는 것 같습니다. 수정하도록 하겠습니다!